src/main/scala/mesosphere/marathon/api/v2/AppsResource.scala

```diff
@@ -315,34 +323,3 @@ class AppsResource @Inject() (
     }
   }
 }
-
-object AppsResource {
-  implicit val WithTaskCountsAndDeploymentsWrites: Writes[AppDefinition.WithTaskCountsAndDeployments] = Writes { app =>
-    val appJson = AppDefinitionWrites.writes(app).as[JsObject]
-
-    appJson ++ Json.obj(
-      "tasksStaged" -> app.tasksStaged,
-      "tasksRunning" -> app.tasksRunning,
-      "tasksHealthy" -> app.tasksHealthy,
-      "tasksUnhealthy" -> app.tasksUnhealthy,
```

Comment:
These `Writes` still exist but have moved to `mesosphere.marathon.api.v2.json.Formats` with the other formatting helpers.